main/src/main/scala/org/clulab/processors/clu/tokenizer/SentenceSplitter.scala

```diff
@@ -119,6 +119,16 @@ abstract class RuleBasedSentenceSplitter extends SentenceSplitter {
         endPositions += lastPosition + raw.head.length
       }
 
+      // found the control string that enforces sentence breaks
+      // note that this token is NOT added to the sentences produced
+      else if(crt.word == SENTENCE_BREAK_CONTROL_STRING) {
```

Comment:
I would be reassured if this was also dependent on `this.useControlStrings` or `SentenceSplitter.useControlStrings` which defaults to `false`.  Those who want to use the feature can turn it on if necessary.

Reply:
Good idea. Can you please add it?

Reply:
No... Because we need to adjust the corresponding antlr grammar. Unless we come up with a generic format for the control string, e.g., anything between square brackets? Or, anything between double square brackets, e.g., [[SB]]? Then we can let people set the string to whatever values they want.